node/src/main/scala/coop/rchain/node/storage/RNodeKeyValueStoreManager.scala

```diff
@@ -0,0 +1,61 @@
+package coop.rchain.node.storage
+
+import java.nio.file.Path
+
+import cats.effect.Concurrent
+import cats.effect.concurrent.{Deferred, Ref}
+import cats.syntax.all._
+import coop.rchain.shared.Log
+import coop.rchain.store.{KeyValueStore, KeyValueStoreManager}
+
+object RNodeKeyValueStoreManager {
+  def apply[F[_]: Concurrent: Log](dirPath: Path): F[KeyValueStoreManager[F]] =
+    Concurrent[F].delay(new RNodeKeyValueStoreManager(dirPath))
+}
+
+private final case class RNodeKeyValueStoreManager[F[_]: Concurrent: Log](
+    dirPath: Path
+) extends KeyValueStoreManager[F] {
+
+  // Database name to store instance name mapping (subfolder for LMDB store)
+  // - keys with the same instance will be in one LMDB file (environment)
+  val instanceDBmapping: Map[String, String] = Map[String, String](
+    // RSpace
+    ("history", "v2/history"),
+    ("cold", "v2/cold"),
+    ("roots", "v2/roots"),
+    // Block store
+    ("block-metadata", "dagstorage")
```

Comment:
`RNodeKeyValueStoreManager` should accept Config and so that this mapping is exactly created from config parameters.